benchmarks/game_of_life_concurrent/procedural_c#/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace procedural_c_
+{
+	class Program
+	{
+		static int runs = 100;
+		static int height = 256;
+		static int width = 256;
+		static int logicalProcessors = Environment.ProcessorCount;
+		static bool[,] board = new bool[width, height];
+
+		static void Main(string[] args)
+		{
+			initilizeBoard();
+			for (int i = 0; i < runs; i++)
+			{
+				updateBoard();
+			}
+			var count = countAlive();
+			Console.WriteLine("Alive: " + count);
+		}
+
+		public static int countLiveNeighbors(int x, int y)
+		{
+			int value = 0;
+			for (int j = -1; j <= 1; j++)
+			{
+				int k = (y + j + height) % height;
+				for (int i = -1; i <= 1; i++)
+				{
+					int h = (x + i + width) % width;
+					if (board[h, k])
+					{
+						value++;
+					}
+				}
+			}
+
+			if (board[x, y])
+				value--;
+			return value;
+		}
+
+
+		public static void updateBoard()
+		{
+			var newBoard = new bool[width, height];
+
+			Parallel.For(0, width, x => {
+				Parallel.For(0, height, y => {
```

Comment:
Vi har tidligere snakket om, at det nok er spild, at det inderste for loop er parallelt, men kan ikke huske, om det giver et performance tab at skrive det